docs/getting-started/guide/cardinality.md

```diff
@@ -0,0 +1,270 @@
+
+## How to add cardinality to your graph
+
+You will need to add a property to your schema that will represent the approximate cardinality of an Entity. This property is usually added to a specific Entity group that exists solely to represent the Cardinality of a given vertex value. An example of the schema changes can be seen in the [advanced properties guide](../../reference/properties-guide/advanced.md#hllsketch). If you are using an Accumulo or Map store as your data store, this should be all that is needed. However, if you are using a custom store, or a custom rest API, some additional config is needed.
+
+!!! tip
+    It is often useful keep track of cardinality per edge group. This is usually done with an edge group property which is group in the groupBy.
+    ``` json
+    {
+      "entities": {
+        "cardinality": {
+          "vertex": "vertex.string",
+          "properties": {
+            "approxCardinality": "hll",
+            "edgeGroup": "set"
+          },
+          "groupBy": [
+            "edgeGroup"
+          ]
+        }
+      }
+    }
+    ```
+
+??? info "Additional config"
+    If you are using a custom data store, you will need to make sure the `SketchesJsonModules` is added to your store properties. This can be easily done by changing the `store.properties` file, as shown below. Alternatively, it can be hardcoded into the store, like in the [AccumuloProperties](https://github.com/gchq/Gaffer/blob/a91ce4cd1e04dd0a2dfdf9633b513768fccd3507/store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloProperties.java#L468).
+    ```
+    gaffer.serialiser.json.modules=uk.gov.gchq.gaffer.sketches.serialisation.json.SketchesJsonModules
+    ```
+
+    If you are using a custom data store, or you not using the standard spring-rest Gaffer rest API, then you will also need to ensure that the `sketches-library` dependency is added to your `pom.xml` for the store and/or rest API.
+    ```xml
+    <dependency>
+        <groupId>uk.gov.gchq.gaffer</groupId>
+        <artifactId>sketches-library</artifactId>
+        <version>${gaffer.version}</version>
+    </dependency>
+    ```
+
+## How to add data with cardinality in your schema
+
+There are two main methods of adding cardinality elements in Gaffer. The first is to do it manually when you add your edges, the second is to use a generator that can do it for you.
+
+### Manually adding cardinality entities
+
+Adding a cardinality entity between vertex 'A' and 'B' using `AddElements`. For more examples of different property types, see the skectehs [dev docs](../../dev/rest-api-sketches.md#primitive-data-types-over-json) page.
+
+=== "Java"
+
+    ``` java
+    final HllSketch hll = new HllSketch(10); // (1)
```

Comment:
Unfortunately, this comment annotation doesn't seem to work when inside the `=== "Java"`, do you know anything about this? @GCHQDeveloper314

Reply:
Fixed it